src/index.js

```diff
@@ -29,26 +50,63 @@ const cors = (options = {}) => handler => (req, res, ...restArgs) => {
     runHandlerOnOptionsRequest = false
   } = options
 
-  res.setHeader('Access-Control-Allow-Origin', origin)
-  if (allowCredentials) {
-    res.setHeader('Access-Control-Allow-Credentials', 'true')
-  }
-  if (exposeHeaders.length) {
-    res.setHeader('Access-Control-Expose-Headers', exposeHeaders.join(','))
+  const requestOrigin = req.headers.origin
+
+  if (!requestOrigin) {
+    return handler(req, res, ...restArgs)
```

Comment:
This might address part of #51, we'll have to make sure that it follows the spec. 

"If the request's origin header is empty, then run the handler."

Reply:
yes, thats exactly what it should do. do we need a test for this?

Reply:
There's no need, I'm going to add a suite of tests making sure we follow the spec.